cogs/announcements.py

```diff
@@ -99,61 +99,60 @@ def __init__(self, bot, utils):
         self.utils.save_settings()
         await context.message.add_reaction('\U0001F44D')
 
+    @commands.command(name='force')
```

Comment:
This command can be included in the @announcement category like all other commands.

Reply:
Is it still commands.has_any_role on the next line?

Reply:
Yep